doc/python_api_reference_vDev.md

```diff
@@ -1040,6 +1041,7 @@ def compile_sampler(
     *,
     skip_reference_sample: bool = False,
     seed: object = None,
+    reference_sample: object = None,
```

Comment:
Make this be `np.ndarray` instead of `object` by adding an `@signature` line to the docstring, similar to how it's done in other methods.

Reply:
When I add a signature and regenerate the docs it doesn't seem to update? Am I missing something simple.

Reply:
Hm, maybe make sure it's the very first line of the docstring? Try copy pasting from another place using `@signature` and confirm it shows that one, so there's as few differences as possible.

This is all just some jank search and replace stuff done in `glue/python/generate_stub_file.py`

Reply:
ugh, I forgot to rebuild the package so naturally import stim.Circuit etc was out of date lol